### DISCUSSION TITLE
System Tray Icon on Windows 11

### DISCUSSION THREAD

**Discussion by 1aTa** [File-New-Project/EarTrumpet#1480] [Ideas]
Windows 11 volume icon on the right and EarTrumpet on the left.

![Screenshot 2023-11-08 092225](https://github.com/File-New-Project/EarTrumpet/assets/37822924/a458ba25-b8f4-4b46-8414-823f4b014566)

Is it possible for the ET icon to match the W11 icon?

**Comment by Hobbes2099** [comment]
I would love if ET could actually **_replace_** the traditional Speaker icon in the System Tray

![image](https://github.com/File-New-Project/EarTrumpet/assets/3689925/9dfbba80-85c5-43c5-b6d0-6d94fbf80120)

**Reply by 1aTa** [reply]
You can use StartAllBack or ExplorerPatcher to do just that, plus a load of other qol improvements.
